pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.tsx

```diff
@@ -54,7 +52,35 @@ export const CategoriesDetailTemplate: React.FC<CategoriesDetailTemplateProps> =
           <LeadParagraph className={styles.description}>{portfolio.longDescription}</LeadParagraph>
         </div>
       )}
-      {component && (
+
+      <div className={styles.resultsDisplaySwitchButtons}>
+        <Button
+          className={styles.buttonIcon}
+          variant={filters.catagoryDisplayLayout === "table" ? "primary-action" : "secondary-action"}
+          onClick={() => setFilters({ ...filters, catagoryDisplayLayout: "table" })}
+        >
+          <FontAwesomeIcon icon={faTable} />
+          {t("Table")}
+        </Button>
+        <Button
+          className={styles.buttonIcon}
+          variant={filters.catagoryDisplayLayout === "cards" ? "primary-action" : "secondary-action"}
+          onClick={() => setFilters({ ...filters, catagoryDisplayLayout: "cards" })}
+        >
+          <FontAwesomeIcon icon={faGripVertical} />
+          {t("Cards")}
+        </Button>
+        <Button
+          className={styles.buttonIcon}
+          variant={filters.catagoryDisplayLayout === "layer" ? "primary-action" : "secondary-action"}
+          onClick={() => setFilters({ ...filters, catagoryDisplayLayout: "layer" })}
+        >
+          <FontAwesomeIcon icon={faLayerGroup} />
+          {t("Layers")}
+        </Button>
+      </div>
```

Comment:
This might be best to be refactored into a separate component, because we're re-using this logic in some other places. Could you create a issue to do this?

Doesn't have to be in this pull request.